.github/workflows/ci.yml

```diff
@@ -3,8 +3,6 @@ name: CI
 
 on:  # cf. https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662
   push:
-    branches:
-      - master
```

Comment:
Why this change?

Isn't it better to only execute this CI pipeline on PRs targetting the `master` branch?